### DISCUSSION TITLE
Command not found after `cargo install sea-orm-cli`

### DISCUSSION THREAD

**Discussion by leo91000** [SeaQL/sea-orm#740] [Q&A] [answered]
Hey !

I tried to install sea-orm-cli following the doc. However, I can't use it after installation :
```shell
cargo install sea-orm-cli
sea-orm-cli generate -h 
  zsh: command not found: sea-orm-cli
```

**Comment by billy1624** [comment, marked as answer]
Hey @leo91000, restart your terminal and try again :)

**Reply by leo91000** [reply]
I needed to add cargo binary path to `.zhsrc`
